Sources/HaCWebsiteLib/NotFoundHandler.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import Kitura
+
+public class NotFoundHandler: RouterMiddleware {
+
+  public func handle(request: RouterRequest, response: RouterResponse, next: @escaping () -> Void) {
+    defer {
+      next()
+    }
+    if response.statusCode != .OK {
```

Comment:
I pulled this down and had a check, errors thrown in routes _do not_ get caught by this, so we'll still need to work on fixing that somehow.

Those errors are much rarer though, so this solves the most pressing case.